recipes/genie/all/conanfile.py

```diff
@@ -14,36 +19,40 @@ class GenieConan(ConanFile):
     description = "Project generator tool"
     topics = ("genie", "project", "generator", "build", "build-systems")
     settings = "os", "arch", "compiler", "build_type"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    win_bash = True
 
     @property
     def _settings_build(self):
+        # TODO: Remove for Conan v2
         return getattr(self, "settings_build", self.settings)
 
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def package_id(self):
+        del self.info.settings.compiler
+
+    def validate(self):
+        if hasattr(self, "settings_build") and cross_building(self):
+            raise ConanInvalidConfiguration("Cross building is not yet supported. Contributions are welcome")
+
     def build_requirements(self):
-        if self.settings.compiler == "Visual Studio":
-            self.build_requires("cccl/1.1")
+        if is_msvc(self):
+            self.build_requires("cccl/1.3")
```

Comment:
```suggestion
            self.tool_requires("cccl/1.3")
```
https://docs.conan.io/en/latest/devtools/build_requires.html#build-and-host-contexts